External/FEXCore/Source/Interface/Core/ArchHelpers/Arm64Emitter.cpp

```diff
@@ -209,19 +209,11 @@ void Arm64Emitter::SpillStaticRegs(bool FPRs, uint32_t GPRSpillMask, uint32_t FP
     }
 
     if (FPRs) {
-      for (size_t i = 0; i < SRAFPR.size(); i+=2) {
```

Comment:
Are these Arm64Emitter changes expected to be done here?
If so, this seems like a de-optimization.

Reply:
If that isn't done, then we'll end up erroneously loading/storing the upper ymm lanes when we shouldn't be, since stp and ldp no longer cross over to the second xmm register

Reply:
Ah right